### ISSUE TITLE
[A11y_Bug]: No information about currently selected date in Calendar for Screen reader user

### ISSUE THREAD

**Issue by satyaminnekanti** [IBMa/equal-access#1898]
### Project

able site

### Browser

Chrome

### Operating System

Windows

### Description

JAWS reads the date and also informs that its selected 

### Steps to reproduce

- Go to [Date Picker live demo](https://carbondesignsystem.com/components/date-picker/usage/#live-demo)
- select a date
- with screen reader ON, Open the date picker and navigate to the selected date

**Expected Result**

JAWS reads the date and also informs that its selected 

**Actual Result**

JAWS just reads the date **BUT no information about the selected state. This is a failure of WCAG [SC 4.1.2:Name, Role, Value](https://www.w3.org/WAI/WCAG22/Understanding/name-role-value.html)**

**Comment by satyaminnekanti** [comment]
sorry wrong repo, closing the isue. logged the issue with carbon-design
